eng/code-quality-reports/src/main/resources/spotbugs/spotbugs-exclude.xml

```diff
@@ -338,4 +338,30 @@
     <Field name="~(MAX_DURATION|NULL_TXN)"/>
     <Bug pattern="MS_SHOULD_BE_FINAL"/>
   </Match>
+
+  <!--  The return value is ignored for purpose-->
```

Comment:
The spacing is off on these comments and the ones below. One space after and one space before.
`<!-- The return value is ignored for purpose -->`

Reply:
fixed